snap/squashfs/squashfs.go

```diff
@@ -509,6 +509,8 @@ type BuildOpts struct {
 // partition table from the snap when a loopback device is created from it. If the snap
 // is smaller than this size, some versions of the kernel will print error logs while
 // scanning the loopback device for partitions.
+// TODO: this isn't reliable, on some distros mkfsquashfs pads up to 64k by
+// default
 const MinimumSnapSize int64 = 16384
```

Comment:
does this mean we should increase this? or those distros make it bigger in mksquashfs but the kernel is happy with smaller anyway?

Reply:
No, AFAIU the default here is fine and correct for upstream squashfs-tools. OpenSUSE chose to patch squashfs-tools to pad to 64k https://build.opensuse.org/package/view_file/filesystems/squashfs/squashfs-64k.patch?expand=1 as a result of this bug https://bugzilla.suse.com/show_bug.cgi?id=953723 and they've been carrying the patch for 8 years now.

Reply:
ok, then the TODO is a big confusing. maybe

// TODO: revisit if necessary, some distros patch user space to pad to 64k but kernel should work with this